src/cpp/include/openvino/genai/speculative_decoding/perf_metrics.hpp

```diff
@@ -0,0 +1,38 @@
+// Copyright (C) 2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#include "openvino/genai/perf_metrics.hpp"
+#include "openvino/genai/visibility.hpp"
+
+
+namespace ov::genai {
+
+struct OPENVINO_GENAI_EXPORTS SDPerfMetrics : public ov::genai::PerfMetrics {
+    ov::genai::MeanStdPair ttst;  // Time to the second token (in ms).
```

Comment:
```suggestion
    ov::genai::MeanStdPair ttst;  // Time from the first to the second token (in ms).
```

Reply:
I have change to `Time of the generation of the second token` 
if I leave this wording here, then `Latency from the third token` should be moved to from the second, but it doesn't sound right, it sounds like time for AVG(2th, 3th .... )